R/f.R

```diff
@@ -6,8 +6,8 @@
 #' @export
 #' @examples
 #' new_f(quote(a))
-#' new_f(quote(a), quote(b))
-new_f <- function(rhs, lhs = NULL, env = parent.frame()) {
+#' new_f(quote(a), lhs = quote(b))
+new_f <- function(rhs, env = caller_env(), lhs = NULL) {
```

Comment:
This new argument order feels a bit weird. Can you comment on why you made the change?